specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-04-01-preview/TenantActivityLogAlerts.json

```diff
@@ -0,0 +1,543 @@
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.AlertsManagement/tenantActivityLogAlerts": {
+      "get": {
+        "description": "Get a list of all Tenant Activity Log Alert rules in the tenant.",
+        "operationId": "TenantActivityLogAlerts_ListByTenant",
+        "x-ms-examples": {
+          "List activity log alerts By Tenant": {
+            "$ref": "./examples/TenantActivityLogAlertRule_ListByTenant.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request succeeded.",
+            "schema": {
+              "$ref": "#/definitions/TenantAlertRuleList"
+            }
+          },
+          "default": {
+            "description": "An error occurred and the list of Activity Log Alert rules could not be retrieved.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{managementGroupName}/providers/Microsoft.AlertsManagement/tenantActivityLogAlerts": {
+      "get": {
+        "description": "Get a list of all Tenant Activity Log Alert rules in a management group.",
+        "operationId": "TenantActivityLogAlerts_ListByManagementGroup",
+        "x-ms-examples": {
+          "List activity log alerts By Management Group": {
+            "$ref": "./examples/TenantActivityLogAlertRule_ListByManagementGroup.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ManagementGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request succeeded.",
+            "schema": {
+              "$ref": "#/definitions/TenantAlertRuleList"
+            }
+          },
+          "default": {
+            "description": "An error occurred and the list of Activity Log Alert rules could not be retrieved.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AzureResource": {
+      "description": "An Azure resource object.",
+      "x-ms-azure-resource": true,
+      "type": "object",
```

Comment:
Instead of defining your own AzureResource its usually recommended to inherit ProxyResource or TrackedResource from common-types.

Reply:
I tried using 'Tracked Resource' at first but I got multiple validation errors for not offering a 'list' operation in the resource group etc. Will you be able to approve despite these errors or should I use anything else? My resource is configured as an extension - and will be allowed to be used as an extension of a management group only - therefore can be listed on a management group only

Reply:
Probably ProxyResource would be the better fit then? If you still have errors about list my resoruce group, I think we could suppress them.